turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
+
+# GCSFuse mount for Turbinia scripts & logging and Google Cloud Functions
+MOUNT_POINT = '/mnt/turbinia'
```

Comment:
Maybe GCS_MOUNT_DIR?

Reply:
Done

Reply:
Looks like it's now GCS_MOUNT_POINT.  Could we s/POINT/DIR/ to make it consistent with the other vars?